doc/source/development/code_style.rst

```diff
@@ -128,6 +128,11 @@ For example:
     value = str
     f"Unknown received type, got: '{type(value).__name__}'"
 
+PEP8 / black
```

Comment:
This is in the wrong section, shouldn't be part of `String formatting.

I think it would make sense to have this as a note at the beginning, than a section of its own.